src/components/infographics.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Infographics
+summary: Infographics should be informative, engaging, and accessible
+version: 0.1.0
+published: false
+accessibility: true
+linkback: https://www.bbc.co.uk/gel/guidelines/how-to-design-infographics
+---
+
+## Introduction
+
+Visual and figurative representations of information can help readers to quickly grasp the salient points. However, without careful and deliberate design they can be confusing, or even misleading. 
+
+The visual design and labeling of infographics (or 'visualisations') is [covered comprehensively in GEL](https://www.bbc.co.uk/gel/guidelines/how-to-design-infographics). As a complement to the GEL documentation, here we shall explore two topics:
+
+1. The technical accessibility requirements of different types of infographic
+2. Alternative and complementary means of consuming the same data/information, provided to uphold the **Offer choice**[^1] inclusive design principle
+
+## Recommended markup
+
+How you encode infographics, and what kind of textual alternatives you provide for them, depends heavily on the nature of the infographic in question. However, it is highly recommended the SVG image format encodes the graphic itself. SVG:
+
+* Is scalable without degradation, facilitating zoom/magnification
+* Can be highly optimized, especially when using the [simple shapes and paths recommended](https://www.bbc.co.uk/gel/guidelines/how-to-design-infographics#infographic-visual-style-overview)
+* Can be maintained and updated easily, like any other markup
+* Unlike runtime-dependent `<canvas>` graphics, can contain accessibility information, and be downloaded as a self-sufficient asset
+
+Inline SVG (SVG embedded directly into the HTML) is the most malleable, since you can target individual SVG elements/artifacts with CSS and JavaScript. Embedding SVG also reduces http requests and, most importantly, _content reflows_[^2], by loading and rendering along with the surrounding content.
+
```

Comment:
Prefer British spelling "artefacts".

Reply:
There are honest differences of opinions about whether image assets in general should be served via separate HTTP requests, because those  requests can be given different caching policies than the text, for example an image could have a 10-year publicly cachable policy based on the filename, whereas a personalised page migh need a no-cache policy. So this argument is nuanced and highly dependent on workflow, content and traffic patterns. I'd recommend sidestepping that debate by not raising the "reduces http requests" angle; The stronger usability argument is that it avoids reflows. In addition, even if a team has strictly separate workflows which demand that designers create images and coders create pages, it's fair to say that data visualisations are a special exception because they are not purely images (they are also data) so they belong _with_ the text.